src/server.h

```diff
@@ -1226,6 +1229,7 @@ struct redisServer {
     struct malloc_stats cron_malloc_stats; /* sampled in serverCron(). */
     redisAtomic long long stat_net_input_bytes; /* Bytes read from network. */
     redisAtomic long long stat_net_output_bytes; /* Bytes written to network. */
+    size_t stat_active_cow_bytes; /* Copy on write bytes while child is active. */
```

Comment:
to comply with the rest of the renames i suggested
```suggestion
    size_t stat_current_cow_bytes; /* Copy on write bytes while child is active. */
```